xwiki-platform-core/xwiki-platform-invitation/xwiki-platform-invitation-ui/src/main/resources/Invitation/Translations.xml

```diff
@@ -42,12 +42,12 @@
   <content>invitation.applicationsPanelEntryLabel=Invitation
 
 # Invitation Administration Section
-Invitation.WebHome_from_address.hint=The email address used to send invitation emails from.
-Invitation.WebHome_smtp_server_password.hint=The password to connect to the SMTP server, only when authentication is required.
-Invitation.WebHome_smtp_server_username.hint=The user name to connect to the SMTP server, only when authentication is required.
+Invitation.WebHome_from_address.hint=The email address used to send the invitation emails.
+Invitation.WebHome_smtp_server_password.hint=The password to connect to the SMTP server.
```

Comment:
I don't really see this as an improvement, an important information has been removed.

Reply:
It will ask for it only when authenticating.

Reply:
That's implied. Password is for authentication.

Reply:
I don't agree, this is not the goal of the task. You are supposed to only work on the English, not change the meaning of a message.

Reply:
..okay